posix-configs/bebop/px4.config

```diff
@@ -34,3 +36,4 @@ mavlink stream -u 14556 -s HIGHRES_IMU -r 50
 mavlink stream -u 14556 -s ATTITUDE -r 50
 df_bebop_bus_wrapper start
 mavlink boot_complete
+sdlog2 start -e -b 20 -t
```

Comment:
Can we start (or should we start) logger here? With a larger buffer (e.g. 200 KB)?